### DISCUSSION TITLE
MY ACTIVITIES DOESNT SHOWING AT THE FEED

### DISCUSSION THREAD

**Discussion by Diegojacober** [community/community#72292] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

 i was showing my profile in other account, and see that my activities like creat repositories or like other repositories aren't showing in my friends feed, WHY??

**Comment by ktauchathuranga** [comment]
have you checked this settings?

![image](https://github.com/community/community/assets/79928743/9d587c0c-67e8-4b13-b9af-b963725357de)
